c2corg_ui/static/partials/card_routes.html

```diff
@@ -1,9 +1,11 @@
-<a ng-click="cardCtrl.openCardPage()" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
   <span class="list-item-title"><span ng-if="locale.title_prefix">{{locale.title_prefix}}&nbsp;: </span>{{locale.title}}</span>
   <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{locale.lang}})</span>
   <br>
   <div class="list-item-info">
-    Min {{doc['elevation_min']}} m / Max {{doc['elevation_max']}} m
+    <span ng-show="doc['elevation_min']">Min {{doc['elevation_min']}} m</span>
+    <span ng-show="doc['elevation_min'] && doc['elevation_max']">/</span>
+    <span ng-show="doc['elevation_max']">Max {{doc['elevation_max']}} m</span>
```

Comment:
OK but don't spend too much time on changes like this. Actually we don't even know if those data are really the ones to display in the cards!